src/components/NumericStepper/NumericStepper.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react';
+import { bem } from '../../utils';
+import styles from './NumericStepper.scss';
+
+import { StepperButton } from '../Buttons';
+
+interface Props {
+    /**
+     * Callback is called when user changes component value by using built-in controls
+     */
+    onChange: (value: number) => void;
+    /**
+     * Increase/decrease step value
+     */
+    step?: number;
+    /**
+     * Minimum possible value of NumericStepper
+     */
+    minValue?: number;
+    /**
+     * Maximum possible value of NumericStepper
+     */
+    maxValue?: number;
+    /**
+     * The value stepping starts from
+     */
+    defaultValue?: number;
+    /**
+     * De-facto CSS property that sets custom width for input
+     */
+    customWidth?: string;
```

Comment:
In string value, we can pass whatever we want and format `3ch` looks complicated. Let's use a number instead of a string.

Reply:
Not really, it's done intentionally, because this value sets input width directly in style attribute. 

It means that instead of 3ch we can use:
1. 100px -> width: 100px
2. 100% -> width: 100%
3.  10vw -> width: 10vw

That's why it still needs to remain a string.

Reply:
Updated the code to eliminate the error message